docs/users/web3.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Use Clerk with Web3
+description: Learn to install and initialize Clerk in a new Web3 application.
+---
+
+# Use Clerk with Web3
+
+Learn how to use Clerk to quickly and easily add secure authentication and user management to your Web3 application.
+
+## Before you start
+
+You need to create a Clerk application in your [Clerk Dashboard](https://dashboard.clerk.com/) and configure the application to use **Sign in with Metamask**. For more information on how to set up a Clerk application, check out the [setup guide](/docs/quickstarts/setup-clerk). To enable Metamask authentication, check out [this guide](/docs/authentication/configuration/sign-in-options#social-o-auth-web3).
+
+## Creating a new Next.js app
+
+Once you have a Clerk application set up in the dashboard, it's time to create a Next.js application. Clerk provides a detailed guide on how to [create a new Next.js app with Clerk](/docs/quickstarts/nextjs).
+
+## Accessing the Web3 address from the frontend
+
+At this point, you should have a Next.js application integrated with Clerk and with Metamask authentication enabled. Run `npm run dev` and visit `localhost:3000`. If you followed the guide, all of the pages to your application should be protected, and you should see your sign-in page. 
+
+Your Clerk application will use the [Account Portal](/docs/account-portal/overview) pages out of the box. However, if you would like more control over your authentication flow, Clerk provides a set of [components](/docs/components/overview) that you can use. If the prebuilt components don't meet your specific needs, you can build fully [custom sign-up and sign-in flows](/docs/custom-flows/overview).
```

Comment:
I feel like this paragraph can be removed, it seems out of place in the context of this content. What do you think?